server/stream.go

```diff
@@ -2450,17 +2455,20 @@ func (mset *stream) setupMirrorConsumer() error {
 		}
 	}
 
-	var filters []string
+	var sfs []string
```

Comment:
Minor nit, but in the case of the length being already known (i.e. `len(mset.cfg.Mirror.SubjectTransforms)`) then it's better to preallocate backing capacity so that `append` won't result in reallocations:
```
sfs := make([]string, 0, len(mset.cfg.Mirror.SubjectTransforms))
```

Reply:
Done